### PR TITLE
Document warning when detecting a CSR clone

### PR DESCRIPTION
**Pull request by deviantintegral**
## Proposed change

This PR has two goals:

1. Make it more likely that if someone searches for "hassio" or "home assistant" plus the error message that they get linked to this page.
2. Give users a hint what to look for if they think they may have unwittingly purchased a cloned adapter.
3. Make it clear that _not_ seeing this message doesn't mean you _don't_ have a clone. I have one I'm 90% sure is, but I don't get this message.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

The error message comes from https://github.com/torvalds/linux/blob/f883675bf6522b52cd75dc3de791680375961769/drivers/bluetooth/btusb.c#L2271-L2272

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by bdraco** [comment]
Will take a look after https://github.com/home-assistant/home-assistant.io/pull/25826 merges

**Comment by deviantintegral** [comment]
Thanks for the link. StarTech USBBT2EDR4

**Comment by deviantintegral** [comment]
Thanks for the link. Of course, I ordered a StarTech USBBT2EDR4 yesterday to see if it's better than the Sabrent one I have that keeps resetting. So, if you end up looking into a way to support those patch files for Broadcom adapters in HASSIO feel free to ping me and I'd be glad to help test.

File: source/_integrations/bluetooth.markdown [hunk 1]
```diff
@@ -194,6 +194,7 @@ These adapters do not have a reset pin. If they stop responding, there is curren
 - tp-link UB500 (RTL8761BU) - Frequent connection failures with active connections
 - Unbranded CSR 4.0 clones with USB id 0a12:0001 - Unrecoverable driver failure
 - 5 CORE CSR 4.0 (CSR CLONE) - Unrecoverable driver failure
+  - These clones will usually show a message like `CSR: Unbranded CSR clone detected; adding workarounds and force-suspending once...` in the system log when they are plugged in.
```
Comment (bdraco) [inline source/_integrations/bluetooth.markdown]:
```suggestion
- CSR 4.0 clones with USB id 0a12:0001 - Unrecoverable driver failure: These clones will usually show a message like `CSR: Unbranded CSR clone detected; adding workarounds and force-suspending once...` in the system log when they are plugged in.
  - Multiple unbranded adapters labeled with CSR 4.0
  - 5 CORE CSR 4.0
```

**Review by bdraco — APPROVED** [review]
Thanks @deviantintegral

**Pull request merged**